config/routes.rb

```diff
@@ -34,7 +34,7 @@
   get '/courses', to: 'courses#index', defaults: { format: :json }
   get "/bib_ids", to: 'courses#bibs', defaults: { format: :json }
 
-  get '/hathi/access', to: 'hathi#hathi_access_bib_status', defaults: { format: :json }
+  get '/hathi/access', to: 'hathi#hathi_access', defaults: { format: :json }
```

Comment:
Can we leave the old route?  Then this could be deployed before the code is updated.  Otherwise we have to update orangelight and requests before this can be deployed.

Reply:
Yes I will add the old route as well but will also keep the new route for the oclc and the bib_id. I will add a comment to remove the old route after we update orangelight and requests with the new route.

Reply:
@carolyncole I added it and removed it again. I don't think the route changes. It is calling a different method but the url is the same. I deployed and tested on bibdata-staging. catalog-staging is looking bibdata-staging. Yesterday we kept the bibdata_base change in the orangelight/requests.yml staging part. I left some commented code. If you think it is ok let me know to remove the code before merging.
The only difference is that the response from the `bib_id` lookup does not include the `id: null` and we would need to update the stubbed urls in requests and orangelight.

Reply:
https://catalog-staging.princeton.edu/catalog/SCSB-3526726
https://catalog-staging.princeton.edu/catalog/SCSB-3923657
https://catalog-staging.princeton.edu/requests/SCSB-3526726?source=pulsearch
https://catalog-staging.princeton.edu/catalog/SCSB-2879206

https://bibdata-staging.princeton.edu/hathi/access?oclc=2243109
https://bibdata-staging.princeton.edu/hathi/access?bib_id=1000046

Reply:
@christinach you are correct.  The route is not changing.  Sorry.

Reply:
No problem @carolyncole thanks you :)